packages/core/docs/vibe-3-changelog.md

```diff
@@ -62,35 +58,48 @@ codemod: `avatar-component-migration`
 ### Dialog
 
 - `shoudlCallbackOnMount` -> `shouldCallbackOnMount` [codemod]
-- `JustifyType` removed [codemod]
+- [!!!] `JustifyType` removed
 
-### DialogContentContainer
+### DialogContentContainer 🚩
 
-- Changed default size to "small" instead of "medium" [codemod - change usages of size medium to small, keep other usages the same]
 - Fixed "medium" size spacings
+- Changed default size to "small" instead of "medium" [codemod - change usages of size medium to small, keep other usages the same]
 
 ### Divider
 
 - `classname` -> `className` [codemod]
 
-### Dropdown
+### Dropdown 🚩
 
 - Removed `Dropdown.size` property, use `Dropdown.sizes` instead [codemod]
 
+### EditableInput
+
+- `EditableInput` removed, use [`EditableText`](https://style.monday.com/?path=/docs/inputs-editabletext--docs) instead
+
+### Heading
+
+- `Heading` component API and style changed - use the [typography migration guide](https://style.monday.com/?path=/docs/typography-migration-guide--docs) to migrate.
+
 ### IconButton
 
 - `dataTestId` -> `data-testid` [codemod]
 
+## Input
+
+- `Input` component removed - use [`TextField`](https://style.monday.com/?path=/docs/inputs-textfield--docs). [codemod]
+
 ### Label
 
 - `wrapperClassName` -> `className` [codemod]
-- Spin in animation removed due to UX decision, including `isAnimationDisabled` prop which is not needed anymore
+- Spin in animation removed due to UX decision
+  - Removed `isAnimationDisabled` prop which is not needed anymore [codemod]
 
 ### EditableHeading
 
 - Complete API Change (TBD)
 
-### Link
+### Link 🚩
```

Comment:
Codemod infra for changing a prop value with static prop namespace
[monday.monday.com/boards/6841644723/pulses/7109091061](https://monday.monday.com/boards/6841644723/pulses/7109091061)

Reply:
Codemod for Link - change Link.target static prop to Link.targets
https://monday.monday.com/boards/6841644723/pulses/7109104112